### ISSUE TITLE
[Bug]: Book match does not work

### ISSUE THREAD

**Issue by h78b7bdfb6737g** [advplyr/audiobookshelf#531]
### Describe the issue

When I try and use the 'Match' function, it never finds the book no matter what service I use or what the book is. 

The debug errors I see are:
[GoogleBooks] Volume search error Error: getaddrinfo EAI_AGAIN www.googleapis.com
[Audible] search error Error: getaddrinfo EAI_AGAIN api.audible.com
[iTunes] search request error Error: getaddrinfo EAI_AGAIN itunes.apple.com
OpenLib Search Error 404

### Steps to reproduce the issue

1. 

### Audiobookshelf version

2.0.3

### How are you running audiobookshelf?

Docker

**Comment by tvories** [comment]
Can you share a screenshot of the match you are trying to make?

**Comment by h78b7bdfb6737g** [comment]
![Alt text](https://i.ibb.co/HTWFCy5/ss.jpg)

**Comment by szemlicka** [comment]
Search works fine for me.

**Comment by advplyr** [comment]
Looks like you may have something blocking the outgoing requests. Search is working well.

**Comment by jrhedman** [comment]
I know this has been closed but I wanted to share my own findings as I'm sure this bug will come up again as I just spent this morning dealing with it.

For my instance, it came from my firewall settings within my Synology NAS where my docker container is running. Until I disabled the firewall I enabled on the NAS. This is also why other people aren't running into this error as it's Synology specific.

I'm not sure what ports I'll need to open on the Synology Firewall yet but once I do, I'll list it here in case other people need this as well.

**Comment by advplyr** [comment]
Great thanks, we can add this to the readme/docs

**Comment by bbrotherton** [comment]
For any fellow wanderers, who are searching for an answer to this issue: I found an easy change that worked for me. I set up a firewall rule to allow traffic from my Docker bridge network access on my Synology NAS.

1. Go to Control Panel > Security > Firewall (tab)
2. Click "Edit Rules" button
3. Click "Create" button to build the rule

I added a rule to allow all ports and all protocols for the IP range that the Docker bridge network uses.
